README.md

```diff
@@ -300,6 +300,31 @@ $client->nextPage($response["pages"]);
 ```
 
 
+## Exceptions
+
+Exceptions are handled by [Guzzle](https://github.com/guzzle/guzzle).
+Intercom Rest API returns `404` HTNL code when a resource is not found in the API (which a common design in REST APIs).
+Guzzle usually throws `404: HTML Not Found` errors when the status code is `404` which is not compatible with our REST API design.
```

Comment:
Tbh I would just say something like: "The Intercom API may return an unsuccessful HTTP response, for example when a resource is not found (404)." Perhaps we can get others to weigh in on this.